app/src/main/java/com/example/lhi06/residue/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+package com.example.lhi06.residue;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+
+public class MainActivity extends AppCompatActivity {
+
+    @BindView(R.id.add_review)
```

Comment:
Actually nothing currently, ive just realised ive actually binded it to the view below